components/FlexibleHeader/src/MDCFlexibleHeaderViewController.m

```diff
@@ -131,22 +124,16 @@ - (void)didMoveToParentViewController:(UIViewController *)parent {
   } else {
     [_headerView trackingScrollViewDidScroll];
   }
+
+  if ([self fhv_shouldUseIdealTopLayoutGuideBehavior]) {
+    [self updateTopLayoutGuide];
+  }
 }
 
 - (void)viewWillAppear:(BOOL)animated {
   [super viewWillAppear:animated];
 
-  for (NSLayoutConstraint *constraint in self.parentViewController.view.constraints) {
```

Comment:
do we want to remove this even if the flag is false?

Reply:
Fixed.